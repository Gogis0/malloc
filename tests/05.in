alloc 293
free 1745
alloc 69
alloc 269
free 1234
alloc 211
free 333
free 282
free 132
free 269
free 1392
free 1699
alloc -26
free 460
alloc 149
free 885
free 1841
free 1405
alloc 1
free 1690
end
