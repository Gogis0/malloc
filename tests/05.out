645
-1
958
1047
-1
1336
-1
-1
-1
-1
-1
-1
-1
-1
1567
-1
-1
-1
1736
-1
